app/src/main/res/layout/notification_center.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout 
```

Comment:
missing constraints?

Reply:
also just curious, is there any reason why you used both constraint and linear layout?

Reply:
Sorry I didn't noticed this one, I thin I will like to go with the linear layout since we are going to implement Toast notification thing cause its simpler and Constraint layout is more complex, but I am open for suggestions